source/Calamari/Commands/RunScriptCommand.cs

```diff
@@ -87,9 +96,15 @@ private int InvokeScript(CalamariVariableDictionary variables)
 
             if (result.ExitCode == 0 && result.HasErrors && variables.GetFlag(SpecialVariables.Action.FailScriptOnErrorOutput, false))
             {
+                if(!(deployment?.SkipJournal ?? true))
```

Comment:
Yes, yes it would...I went through a couple of different options but totally missed that one 😳